.travis.yml

```diff
@@ -36,6 +36,11 @@ env:
     - BUILD_PROCESSOR="`uname -p`"
     - OPENMVG_SOURCE=${TRAVIS_BUILD_DIR}/src
     - OPENMVG_BUILD=${TRAVIS_BUILD_DIR}/build
+    # CMAKE
+    - CMAKE_URL="https://cmake.org/files/v3.4/cmake-3.4.1-Linux-x86_64.tar.gz"
```

Comment:
ok, but it won't be a problem to use a different one, as the PopsiftConfig generate something compatible with >= 2.8.12